packages/twenty-front/src/modules/object-metadata/schemas/createObjectMetadataItemPayloadSchema.ts

```diff
@@ -0,0 +1,31 @@
+import camelCase from 'lodash.camelcase';
+import { z, ZodTypeDef } from 'zod';
+
+import { objectMetadataItemSchema } from '@/object-metadata/schemas/objectMetadataItemSchema';
+import { CreateObjectInput } from '~/generated-metadata/graphql';
+
+type Input = Omit<
```

Comment:
Why do you need to first omit a few fields in the `Input` type and then only pick the one you want in the `schema`? Doing the picking in the schema would not be enough?

Reply:
I had defined separate `Input` and `Output` types to be able to use `z.input<typeof createObjectMetadataItemPayloadSchema>` easily (which returns the `Input` type of a schema), but after discussing about this with Charles, this might not be necessary anymore. I removed the `Input`/`Output` types.